test/python/WMCore_t/WMBS_t/JobSplitting_t/FileBased_t.py

```diff
@@ -129,7 +129,7 @@ def createLargeFileBlock(self):
         """
         testFileset = Fileset(name = "TestFilesetX")
         testFileset.create()
-        for i in range(5000):
+        for _ in range(5000):
```

Comment:
I don't know. I just tried to keep the logic exactly the same. 

The 5 or 10 minutes isn't *really* a problem, it just sets the scale of how short you can make the total testing cycle if you farm things out more widely.